src/pages/Settings/PaymentSettings.vue

```diff
@@ -24,7 +24,7 @@
 								class="payment-settings-default-form__cc-wrapper"
 							>
 								<kv-radio
-									class="payment-settings-default-form__radio"
+									class="payment-settings-default-form__radio fs-mask"
```

Comment:
I think we should either exclude all of this or wrap `paymentMethod.description` in a span and exclude just that if we want to see the payment type icon.